adamsjr8576/palette-picker-fe
# Testing fe app

### What does this PR do?
- Tests actions, reducers, ProjectCard, ProjectPalette, and add test to PaletteCard.
- gets actions and reducers at 100% coverage and gets all components over 90%
- uses react-resting-library for components

### How to test?
- Pull down
- run `npm test`
- can also run `npm test --  --coverage --watchAll=false` to see test coverage

### Issues
- This PR closes no issues

<img width="693" alt="Screen Shot 2020-02-11 at 10 08 44 PM" src="https://user-images.githubusercontent.com/49846853/74304923-9eafd280-4d1b-11ea-9b39-91f661ec5475.png">
